### PR TITLE
create a venv script to create and automatically link a ven to a kernel

### PR DESCRIPTION
**Pull request by 12rambau**
- create a venv script 
- sanityse all the other script (because they were simply ugly)
- add docstring everywhere to allow the `--help` call
- update documentation

### CODE REVIEW

File: sepal_ui/bin/module_venv [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    res = subprocess.run(cmd, cwd=Path.cwd())
+
+    cmd = [pip, "install", "Cython"]
+    res = subprocess.run(cmd, cwd=Path.cwd())
+
+    cmd = [pip, "install", "ipykernel"]
+    res = subprocess.run(cmd, cwd=Path.cwd())
+
+    # install all the requirements
+    cmd = [pip, "install", "-r", str(Path.cwd() / "requirements.txt")]
```
Comment (dfguerrerom) [inline sepal_ui/bin/module_venv:68]: Can we merge the cmd's and loop the `run` method? perhaps we can save some lines :)

File: sepal_ui/bin/module_venv [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    print(f"{Fore.YELLOW}venv creation interface{Fore.RESET}")
+
+    # create a venv folder
+    print('create the venv directory: "module-venv"')
+    venv_dir = Path.home() / "module-venv"
+    venv_dir.mkdir(exist_ok=True)
+
+    # create a venv folder associated with the current repository
+    print(f'create a venv directory for the current app: "{Path.cwd().stem}"')
+    current_dir_venv = venv_dir / Path.cwd().stem
```
[the remark this answers is not in the document]
Comment (12rambau) [reply sepal_ui/bin/module_venv:46]: in the end I don't care if we are in a git repository, I just need to make sure that it's a module. My guess is that nobody use "ui.ipynb" as a notebook name

**Comment by dfguerrerom** [comment]
We are not installing the new scripts in the setup, so can't be accessed from the console, is there something else I have to do to test without calling directly in its directory? i.e.: 

`$ module_deploy` ...

**Comment by 12rambau** [comment]
> We are not installing the new scripts in the `setup.py` file, so can't be accessed from the console, is there something else I have to do to test without calling directly in its directory? i.e.:
> 
> `$ module_deploy` ...

I have definitely added the sepal_ui/bin folder to `$Path`, I though Daniel W. had done the same for all SEPAL users (https://github.com/openforis/sepal/issues/73). It seems it's not the case. Do you know a way to add this script via setup.py ? 
if yes I'm interested !

**Comment by dfguerrerom** [comment]
> > We are not installing the new scripts in the `setup.py` file, so can't be accessed from the console, is there something else I have to do to test without calling directly in its directory? i.e.:
> > `$ module_deploy` ...
> 
> I have definitely added the sepal_ui/bin folder to `$Path`, I though Daniel W. had done the same for all SEPAL users ([openforis/sepal#73](https://github.com/openforis/sepal/issues/73)). It seems it's not the case. Do you know a way to add this script via setup.py ? if yes I'm interested !

The last commit works for me.

**Review by dfguerrerom — APPROVED** [review]

**Pull request merged**